cli/popper/commands/cmd_add.py

```diff
@@ -48,9 +48,12 @@ def cli(ctx, pipeline):
 
     pu.info("Downloading pipeline {} ... ".format(pipeline_name))
     r = requests.get(download_url)
-    z = zipfile.ZipFile(BytesIO(r.content))
-    z.extractall()
-    z.close()
+    try:
+        z = zipfile.ZipFile(BytesIO(r.content))
+        z.extractall()
+    except BadZipFile as e:
+        z.close()
```

Comment:
in the case of an exception, this will close the file and keep going. Should we instead throw an error when `BadZipFile` is raised?

Also, we might want to check the code we get in line 50 so that we are certain that the request succeeded

Reply:
> in the case of an exception, this will close the file and keep going. Should we instead throw an error when BadZipFile is raised?

I am not sure if we would need to do this since it is not interfering with the downloads. What do you think?